lib/service/service.go

```diff
@@ -666,7 +676,6 @@ func NewTeleport(cfg *Config) (*TeleportProcess, error) {
 	}
 
 	if cfg.Proxy.Enabled {
-		eventMapping.In = append(eventMapping.In, ProxySSHReady)
```

Comment:
This is removed but doesn't appear to be re-added elsewhere - was it redundant/incorrect code?

Reply:
This is a duplicate, event already added on line 653 above